WordPress/Classes/Utility/In-App Feedback/InAppFeedbackPromptCoordinator.swift

```diff
@@ -0,0 +1,172 @@
+import Foundation
+import StoreKit
+
+protocol InAppFeedbackPromptPresenting {
+    func presentIfNeeded(in controller: UIViewController) -> Bool
+}
+
+struct InAppFeedbackPromptCoordinator: InAppFeedbackPromptPresenting {
+
+    // MARK: - Properties
+
+    private let appRatingUtility: AppRatingUtility
+    private let userDefaults: UserDefaults
```

Comment:
`UserPersistentRepository` can be used instead of `UserDefaults` as type. It simply adds an abstraction over `UserDefaults` to allow us injecting.

Reply:
Resolved in 1162ccbfdb02c4a7af7616bcd5315847e425d769